regression/Makefile

```diff
@@ -10,6 +10,7 @@ DIRS = ansi-c \
        goto-instrument \
        goto-instrument-typedef \
        test-script \
+       ansi-c-typename \
```

Comment:
I'd just add it to the `goto-instrument` directory.

Reply:
done